.github/workflows/docker-lint.yml

```diff
@@ -0,0 +1,38 @@
+name: Docker Lint
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: read
+
+concurrency: 
+  group: docker-lint-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  docker-lint:
+    runs-on: ubuntu-latest
+
+    name: docker-lint
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: install npm
+        run: |
+          sudo apt-get install nodejs npm -y
```

Comment:
why not https://github.com/actions/setup-node?

Reply:
Good point, thanks!

Reply:
This broke the logic so reverted for now.